cli/flox-rust-sdk/src/utils/mod.rs

```diff
@@ -158,7 +158,7 @@ impl Display for DisplayCommand<'_> {
 }
 
 /// Returns a `tracing`-compatible form of a [Path]
-pub fn traceable_path(p: &impl AsRef<Path>) -> impl tracing::Value {
+pub fn traceable_path(p: impl AsRef<Path>) -> impl tracing::Value {
```

Comment:
The `&` isn't necessary because if `T: AsRef<U>` then so does `&T`.

tl;dr you don't need as many `&`s at the call site in some cases and retain exactly the same behavior.